### PR TITLE
Front page tweaks for 2019

### PR DESCRIPTION
**Pull request by shanecoughlan**
Adjusted weird forwardslash,backslash in </a> tags in JP version (i.e. <\/a> to end all tags.

Adjusted one case copy/pasted to EN version in LEARN MORE CONTENT that may have been the cause of our break.

### CODE REVIEW

File: WebContent/resources/locales/ja/translation.json [hunk 1]
```diff
@@ -51,13 +51,13 @@
...
-  "footer_content":"Copyright 2018 The Linux Foundation. 無断複写・転載を禁じます。Linux Foundation は登録商標を持ち、商標を使用します。Linux Foundationの登録商標リストについては、<a class=\"anchor\" href=\"https:\/\/www.linuxfoundation.org\/trademark-usage\" target=\"_blank\">登録商標の使用<\/a>のページをご覧ください。LinuxはLinus Torvaldsの登録商標です。<a class=\"anchor\" href=\"https:\/\/www.openchainproject.org\/privacy\">プライバシーポリシー<\/a>および<a class=\"anchor\" href=\"https:\/\/www.openchainproject.org\/terms\">利用規約<\/a>",
+  "Please contact":"エラーや問題がある場合は<a class=\"anchor\" href=\"mailto:openchain-conformance@lists.linuxfoundation.org?Subject=OpenChain%20Certification%20Feedback\" target=\"_blank\">OpenChain サポート</a>に連絡してください。",
+  "footer_content":"Copyright 2018 The Linux Foundation. 無断複写・転載を禁じます。Linux Foundation は登録商標を持ち、商標を使用します。Linux Foundationの登録商標リストについては、<a class=\"anchor\" href=\"https:\/\/www.linuxfoundation.org\/trademark-usage\" target=\"_blank\">登録商標の使用</a>のページをご覧ください。LinuxはLinus Torvaldsの登録商標です。<a class=\"anchor\" href=\"https:\/\/www.openchainproject.org\/privacy\">プライバシーポリシー</a>および<a class=\"anchor\" href=\"https:\/\/www.openchainproject.org\/terms\">利用規約</a>",
```
Comment (goneall) [inline WebContent/resources/locales/ja/translation.json]: It looks like there are some backslashes in the href's as well.

**Comment by goneall** [comment]
I went back and lookated at the history and the backslashes were in the original japanese translation file, so that may not be the cause of the recent issues

**Comment by goneall** [comment]
The changes conflict with the latest version which will need to be resolved before merging.

**Comment by shanecoughlan** [comment]
@goneall I think I fixed the merge conflict. Can we give it a shot?

**Comment by goneall** [comment]
This is the commit that caused the issue with Japanese no longer working: https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/commit/f6aa614b8da744aa43189c563f4ea4f20c9e3d4b

**Comment by goneall** [comment]
I tested with this branch and it is still broken.

**Comment by goneall** [comment]
Found the issue and fixed it with PR #166 

This has been deployed

**Pull request closed without merging**